src/components/oauth/OAuthErrorHandler.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @author sarahr
+ */
+
+import React from "react";
+
+import { useTranslation } from "i18n";
+import ErrorIcon from "@material-ui/icons/Error";
+import { build } from "@cyverse-de/ui-lib";
+import {
+    Card,
+    CardActions,
+    CardContent,
+    CardHeader,
+    Divider,
+    Grid,
+    makeStyles,
+    Typography,
+} from "@material-ui/core";
+
+import ClientInfo from "../utils/error/ClientInfo";
+import ContactSupport from "../utils/error/ContactSupport";
+import ids from "./ids";
+
+const useStyles = makeStyles((theme) => ({
+    cardContent: {
+        padding: theme.spacing(2),
+        [theme.breakpoints.down("xs")]: {
+            padding: theme.spacing(1),
+        },
+    },
+    container: {
+        margin: theme.spacing(1),
+        [theme.breakpoints.down("xs")]: {
+            margin: theme.spacing(0),
+        },
+    },
+}));
+
+function OAuthErrorHandler(props) {
+    const { errorCode, baseId } = props;
+    const { t } = useTranslation("oauth");
+    const errorBaseId = build(baseId, ids.OAUTH_ERROR_HANDLER);
+    const classes = useStyles();
+
+    const avatar = <ErrorIcon fontSize="large" color="error" />;
+    const title = (
+        <Typography color="error" variant="h6">
+            {t("oops")}
+        </Typography>
+    );
+    const subheader = <Typography color="error">{t(errorCode)}</Typography>;
+
+    return (
+        <Card id={errorBaseId}>
+            <CardHeader avatar={avatar} title={title} subheader={subheader} />
```

Comment:
FYI, the `CardHeader` component accepts props `titleTypographyProps` and `subheaderTypographyProps`, since it's already wrapping the `title` and `subheader` in its own `Typography` components.

That means this text will be double-wrapped in `Typography` components (which probably isn't a big deal).

This code looks pretty tidy, though, so I'll leave it to personal preference, but here's an example where I'm using `titleTypographyProps`:
https://github.com/cyverse-de/sonora/blob/3d8cb7640ef34c6192d8559f8d33f7fbbc48931f/src/components/apps/AppStepDisplay.js#L58-L65